test/TokenRequestTest.js

```diff
@@ -1,51 +1,304 @@
-const TokenRequest = artifacts.require('TokenRequest')
-const TokenManager = artifacts.require('TokenManager')
+const { encodeCallScript } = require('@aragon/test-helpers/evmScript')
+const getBalanceFn = require('@aragon/test-helpers/balance')
 
 import DaoDeployment from './helpers/DaoDeployment'
-import { deployedContract } from './helpers/helpers'
+import { deployedContract, assertRevert } from './helpers/helpers'
+const ForwarderMock = artifacts.require('ForwarderMock')
+const MiniMeToken = artifacts.require('MiniMeToken')
+const MiniMeTokenFactory = artifacts.require('MiniMeTokenFactory')
+const MockErc20 = artifacts.require('TokenMock')
+const TokenManager = artifacts.require('TokenManager')
+const TokenRequest = artifacts.require('TokenRequest')
+const Vault = artifacts.require('Vault')
+
+contract('TokenRequest', ([rootAccount, ...accounts]) => {
+  let daoDeployment = new DaoDeployment()
+  let requestableToken,
+    tokenRequestBase,
+    tokenRequest,
+    tokenManager,
+    tokenManager2,
```

Comment:
If it's only used in the 'setTokenManager' describe, it could be declared there and not globally.